### DISCUSSION TITLE
Existing Map - Offline Access

### DISCUSSION THREAD

**Discussion by LanceMcDermott** [komoot/photon#805] [General]
Currently ingesting all the data, but curious to what url should be used in place of say https://nominatim.openstreetmap.org/search? Currently using this to run nominatim in the software, but swapping to Photon for offline access and curious to what should go in place of the above url? http://localhost:2322/api?

**Comment by mtmail** [comment]
Yes, http://localhost:2322/api? (https://github.com/komoot/photon?tab=readme-ov-file#search-api). Remember some URL parameters are also different between Nominatim and Photon (the `q` is the same).
